pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2778 @@
+/*
```

Comment:
@feiskyer I think @vishh means before build, it's possible to run the code generation script to get the `.go` file, so you have to check in the `.go` in commits.
But I am ok with both, not a big deal.